src/openzaak/components/autorisaties/openapi.yaml

```diff
@@ -111,13 +111,13 @@ paths:
       - name: page
         required: false
         in: query
-        description: Een pagina binnen de gepagineerde set resultaten.
```

Comment:
These are added in https://github.com/open-zaak/open-zaak/pull/1898/files, didn't want to cause merge conflicts with the translation files